src/gui/status_footer.cpp

```diff
@@ -92,7 +92,7 @@ void status_footer_init(status_footer_t *footer, int16_t parent) {
         &(footer->wt_z_axis));
     footer->wt_z_axis.font = resource_font(IDR_FNT_SPECIAL);
     window_set_alignment(id, ALIGN_CENTER);
-    window_set_text(id, "999.95");
+    window_set_text(id, "0.00");
```

Comment:
is 0.00 a valid value or an estimate for the Z axis' position? Wouldn't a `"0.00?"` reflect the situation better?